TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
@ericstj This is a library referenced by a runnable application. Does it still hold true that this should be netcoreapp1.0?